build/commands/scripts/commands.js

```diff
@@ -87,6 +87,14 @@ program
   .option('--bitflyer_staging_client_id <bitflyer_staging_client_id>')
   .option('--bitflyer_staging_client_secret <bitflyer_staging_client_secret>')
   .option('--bitflyer_staging_url <bitflyer_staging_url>')
+  .option('--gemini_api_url <gemini_api_url>')
```

Comment:
@zenparsing not sure if this is what you're asking, but I'm wondering if we really need all of these. Also the options in https://github.com/brave/brave-core/pull/8566/files#diff-c762921b1126e4475bffab885d122da3939e98234baa81d784b613ee352bc9cdR134 don't match so that would force a full rebuild. We need to stop copying options like this across multiple commands, it's extremely fragile and error prone. I'd rather make these npm only configs